src/convertToObject.js

```diff
@@ -6,7 +6,32 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  // Split the input string by semicolon to separate each CSS declaration
+  const declarations = sourceString.split(';');
+
+  for (const declaration of declarations) {
+    // Ignore empty declarations
+    if (!declaration.trim()) {
+      continue;
+    }
+
+    // Split each declaration by colon to separate the property and value
+    const [property, value] = declaration.split(':');
```

Comment:
Consider using `declaration.split(':', 2)` to ensure that only the first colon is used to split the property and value. This will handle cases where the value itself contains colons.